src/activities/components/ActivityList.vue

```diff
@@ -334,9 +334,50 @@ export default {
       icsDialog: false,
       numDisplayed: NUM_ACTIVITIES_PER_LOAD,
       types: [],
+      activityListTypeFilter: this.filterActivityTypes.slice(),
```

Comment:
I think this is not needed here, and I think it'll also prevent `activityListTypeFilter` being reactive to changes to the prop value.